src/DynamoCore/Graph/Nodes/DummyNode.cs

```diff
@@ -365,14 +365,14 @@ private XmlElement OriginalXmlNodeContent
         {
             get
             {
-              if (OriginalNodeContent == null)
-                  return null;
+                if (OriginalNodeContent == null)
+                    return null;
 
-              XmlElement originalXmlElement = OriginalNodeContent as XmlElement;
-              if (originalXmlElement == null)
-                  throw new InvalidOperationException("OriginalNodeContent is not an XmlElement.");
+                XmlElement originalXmlElement = OriginalNodeContent as XmlElement;
+                if (originalXmlElement == null)
+                    return null;
 
-              return originalXmlElement;
+                return originalXmlElement;
```

Comment:
@mjkkirschner This is the fix I put in to deal with this issue.

Reply:
So this is obviously not related to Dictionary changes. We could submit this immediately in a separate PR?

Reply:
@ColinDayOrg fyi